core/web/presenters/job.go

```diff
@@ -168,31 +168,31 @@ func NewOffChainReportingSpec(spec *job.OffchainReportingOracleSpec) *OffChainRe
 
 // OffChainReporting2Spec defines the spec details of a OffChainReporting2 Job
 type OffChainReporting2Spec struct {
-	ContractAddress                        ethkey.EIP55Address  `json:"contractAddress"`
-	P2PPeerID                              *p2pkey.PeerID       `json:"p2pPeerID"`
-	P2PBootstrapPeers                      pq.StringArray       `json:"p2pBootstrapPeers"`
-	IsBootstrapPeer                        bool                 `json:"isBootstrapPeer"`
-	EncryptedOCRKeyBundleID                null.String          `json:"keyBundleID"`
-	TransmitterAddress                     *ethkey.EIP55Address `json:"transmitterAddress"`
-	ObservationTimeout                     models.Interval      `json:"observationTimeout"`
-	BlockchainTimeout                      models.Interval      `json:"blockchainTimeout"`
-	ContractConfigTrackerSubscribeInterval models.Interval      `json:"contractConfigTrackerSubscribeInterval"`
-	ContractConfigTrackerPollInterval      models.Interval      `json:"contractConfigTrackerPollInterval"`
-	ContractConfigConfirmations            uint16               `json:"contractConfigConfirmations"`
-	CreatedAt                              time.Time            `json:"createdAt"`
-	UpdatedAt                              time.Time            `json:"updatedAt"`
+	ContractAddress                        null.String     `json:"contractAddress"`
```

Comment:
I wonder if this belongs in the relay json

Reply:
Possible, as for example for Solana we need 3x contract address/accout (the program itself, program state, transmissions state) while just the contract address is enough for Ethereum.

The current flow for Solana takes the program account/address from the spec itself, and then the other two from the `RelayConfig` JSON part of the spec.

But as we are rushing, we can just go like this for now and add a TODO comment that it could make sense to move it.

Reply:
this should just be contractID, meant to come back to it. Updated and included the rest of the new fields